notebooks/Pipeline Scoring Example.ipynb

```diff
@@ -363,49 +311,50 @@
    "metadata": {},
    "source": [
     "Afterwards, we pass the ground truth, the detected anomalies and the original data\n",
-    "to the `orion.metrics.accuracy_score` and `orion.metrics.f1_score` functions in order\n",
+    "to the `orion.metrics.contextual_accuracy` and `orion.metrics.contextual_f1_score` functions in order\n",
```

Comment:
Do we have a document introducing the definitions of the metrics? If yes, we can provide the links here.

Reply:
Done!